core/src/scss/core/helpers/_aspect-ratio.scss

```
@@ -16,10 +16,10 @@
   @include aspect-ratio(16, 9);
 }
 
-.su-aspect-ratio-4x3 {
+.su-aspect-ratio--4x3 {
   @include aspect-ratio(4, 3);
 }
 
-.su-aspect-ratio-1x1 {
+.su-aspect-ratio--1x1 {
```

```suggestion
.su-aspect-ratio-1x1,
.su-aspect-ratio--1x1 {
```

Backwards compatibility with version release.